prober/derp.go

```diff
@@ -47,8 +54,20 @@ type derpProber struct {
 	probes        map[string]*Probe
 }
 
+type DERPOpt func(*derpProber)
+
+// WithBandwidthProbing enabled optional bandwidth probing. When enabled,
+// a payload of `size` bytes will be regularly transferred through each
+// DERP server, and each pair of DERP servers in every region.
+func WithBandwidthProbing(interval time.Duration, size int64) DERPOpt {
+	return func(d *derpProber) {
+		d.bwInterval = interval
+		d.bwProbeSize = size
+	}
+}
+
 // DERP creates a new derpProber.
-func DERP(p *Prober, derpMapURL string, udpInterval, meshInterval, tlsInterval time.Duration) (*derpProber, error) {
+func DERP(p *Prober, derpMapURL string, udpInterval, meshInterval, tlsInterval time.Duration, opts ...DERPOpt) (*derpProber, error) {
```

Comment:
Now that we're adding options, maybe we can make those other intervals into opts too? Or are those required?

(I don't _love_ the variadic opts pattern, but here it seems sorta OK, and if we're going to have it anyway we might as well lean on it)

Reply:
Moved all intervals to options.